run.go

```diff
@@ -23,17 +23,24 @@ func Run(f func(ctx context.Context) error) {
 	ctx := trap.Context()
 
 	// Handle uncaught panics.
+	errMsg := fmt.Sprintf("View the task logs for more details: app.airplane.dev/runs/%s", os.Getenv("AIRPLANE_RUN_ID"))
```

Comment:
minor, could be shorter:

```suggestion
	errMsg := fmt.Sprintf("View the task logs for details: app.airplane.dev/runs/%s", os.Getenv("AIRPLANE_RUN_ID"))
```